docs/aad-e2e/aad-e2e.md

```diff
@@ -0,0 +1,246 @@
+### Integrate Azure Active Directory (AAD) with OpenPAI
+
+Start from version 0.15 OpenPAI can be configured to use Azure Active Directory (AAD) for user authentication. In this configuration, you can sign in to an OpenPAI cluster by using your AAD authentication token.
+
+This document will cover how to set up the integration step by step.
+#### Note
+
+- If integrate AAD with OpenPAI, the user management will be taken over by AAD. The local user data will be useless
```

Comment:
Previous user data in webportal is required to be mapping/migrate to AAD. Once the integration is enabled, Instead of using built-n user managed webportal, OpenPAI will switch to use (and only use) AAD as user authentication mechanism.